### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by TomS-** [rstacruz/jquery.transit#241]
I've been using AnimeJS however it doesn't have a way of returning a value or applying a value without a delay. 

These are two I need for the current project I have. Transform properties are a nightmare unless using something like this. I can't find anything that replaces this, however bugs such as not being able to get scale() it always returns 1 regardless is meaning I can't use it. 

Is this dead? Can I fork it and start from where you left off?

**Comment by Mohsen-Khakbiz** [comment]
did you try TweenMax?

**Comment by TomS-** [comment]
Yes, however I prefer AnimeJS (11kb) to TweenMax (111kb) due to the size difference.

**Comment by quietTrojan** [comment]
can you recommend similar project ?
